Mu2eG4/src/physicsListDecider.cc

```diff
@@ -122,36 +115,30 @@ namespace mu2e{
       tmpPL->RegisterPhysics(new G4RadioactiveDecayPhysics(debug.diagLevel()));
     }
 
-#if G4VERSION>4103
-    // for version 4105 it will need to be rplaced with
-    // emParams->SetMscEnergyLimit(115.0*CLHEP::MeV);
-    if ( phys.modifyEMOption4() && (name.find("_EMZ") != std::string::npos) ) {
-      tmpPL->RemovePhysics(("G4EmStandard_opt4"));
+#if G4VERSION>4104
+
+    // changing msc model transition energy if requested
+    double mscTransitionEnergyValue;
```

Comment:
Please initialize mscTransitionEnergyValue.

Reply:
I have mixed feelings about initializing something if it is not necessarily going to be used, or if the initial value can cause problems, so I added a scope to prevent its use outside of it;

Reply:
> I have mixed feelings about initializing something if it is not necessarily going to be used [...]

If it is really not used then the initializer value does not matter; just assign zero or max() as you did,  but I think it is good to have deterministic code behavior.   I remember hearing (a while ago) about getting floating point exceptions because occasionally an ununitialized floating var would get a signalling NaN as its content by chance.  The code logic was correct and the ununitialized value was not used  in computations, but still caused problems when objects containing uninitialized varialbles were copied.   I think it is a good practice to require all variables in our "normal" code to be initialized at the point of declaration, as many C++ guides suggest.   (There are exceptions to always initialize to gain some efficiency; I think the STL uses uninitialized memory internally.  But this is not for user level code.)